Library/Homebrew/test/bottle_hooks_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "formula_installer"
+require "hooks/bottles"
+
+describe Homebrew::Hooks::Bottles do
+  subject { formula_installer }
+
+  let(:formula_installer) {
```

Comment:
How about doing `let(:formula_installer) { FormulaInstaller.new formula }` for one-liners < 80 characters?